File: tasks/2024/round3_pp/hetkrajcar.cpp
// @check-accepted: examples Teljes
// insert brief description of the solution here

#include <fstream>
#include <iostream>
#include <vector>

using namespace std;

int main() {
    // uncomment the following lines if you want to read/write from files
    // ifstream cin("input.txt");
    // ofstream cout("output.txt");
    int N, t;

    cin >> N;
    for(int i = 1; i <= N; i++)
    {
    	cin >> t;
    	if(t==7)
    	{
    		cout << i << "\n";
    		return 0;
    	}
    }
    cout << -1 << "\n";
    return 0;
}

File: tasks/2024/round3_pp/hetkrajcar.py
#!/usr/bin/env python3
# @check-accepted: examples Teljes
# insert brief description of the solution here


# input data

# insert your code here
N =  int(input())

T = list(map(int, input().split()))
ans=-1
for i in range(0,N):
 	if T[i] == 7:
    		ans=(i+1)
    		break
print(ans)
